packages/AdaptiveCards/Microsoft.Bot.Components.AdaptiveCards.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Bot.Components.AdaptiveCard</PackageId>
+    <RepositoryUrl>https://github.com/microsoft/botframework-components/tree/main/packages/AdaptiveCards</RepositoryUrl>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <Description>Composer triggers and actions for working with Adaptive Cards.</Description>
+    <Summary>Composer triggers and actions for working with Adaptive Cards.</Summary>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\build\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>true</DelaySign>
+    <PackageTags>msbot-component;msbot-action;msbot-trigger</PackageTags>
```

Comment:
`msbot-component` is included in Directory.Build.props, it can be removed (the others can stay).

Reply:
got it

---
In reply to: [609116536](https://github.com/microsoft/botframework-components/pull/862#discussion_r609116536) [](ancestors = 609116536)